django/template/base.py

```diff
@@ -880,6 +880,9 @@ def _resolve_lookup(self, context):
         try:  # catch-all for silent variable failures
             for bit in self.lookups:
                 try:  # dictionary lookup
+                    # Only allow if the metaclass implements __getitem__.
```

Comment:
I would be inclined to add a docs links here, likely: https://docs.python.org/3/reference/datamodel.html#classgetitem-versus-getitem

It took me a while to find docs explaining *exactly* what was going one here. What do you think?